en/config/actuators.md

```diff
@@ -1,139 +1,484 @@
+Motors and actuators (which are referred to as "functions") can be assigned to any physical output on any of the available busses.
 
 :::note
-For boards with MAIN and AUX, prefer the AUX pins over the MAIN pins for motors, as they have lower latency.
+PWM AUX outputs are preferred over the PWM MAIN outputs for controlling motors (they have lower latency).
 :::
 
-The AUX pins have additional configuration options for camera capture/triggering.
+PWM outputs are grouped based on the protocols that they support.
+In other words, the outputs that support DShot/Oneshot/PWM would be grouped together, separate from those that don't support Dshot (say).
+
+The AUX tab also has "CAP" options for camera capture/triggering.
 Selecting these requires a reboot before they are applied.
 
+You should assign functions to the outputs that match your physical wiring of motors and servos, and use the [Actuator Testing](#actuator-testing) section described below to determine appropriate output parameter values.
+These steps are covered in [Output Assignment and Configuration](#output-assignment-and-configuration).
 
 ## Actuator Testing
 
-<!-- Not documented: "Identify and assign motors" -->
- 
-When testing actuators, make sure that:
-- Motors spin at the "minimum thrust" position.
+The _Actuator Testing_ section in lower-right corner provides sliders that can be used to test (and determine) actuator and motor settings. 
+A slider is provided for each output defined in the [Actuator Outputs](#actuator-outputs) section.
+The slider example below shows the section for a typical VTOL Tiltrotor airframe.
 
-  The sliders snap into place at the lower end, and motors are turned off (disarmed).
-  The "minimum thrust" position is the next slider position, which commands the minimum thrust.
-  For PWM motors, adjust the minimum output value such that the motors spin at that slider position (not required for DShot).
-  :::note
-  VTOLs will automatically turn off motors pointing upwards during fixed-wing flight.
-  For Standard VTOLs these are the motors defined as multicopter motors.
-  For Tiltrotors these are the motors that have no associated tilt servo.
-  Tailsitters use all motors in fixed-wing flight.
-  :::
-- Servos move into the direction of the convention described above.
-  :::note
-  A trim value can be configured for control surfaces, which is also applied to the test slider.
-  :::
+![Actuator Testing Slider](../../assets/config/actuators/vtol_tiltrotor_sliders_example.png)
+
+The section has an **Enable Sliders** switch that must be toggled before sliders can be used.
+The sliders can power the motors/servos across their full range of motion, and "snap" to the disarmed and minimum positions.
+
+:::note
+After you toggle the **Enable sliders** switch, actuators/motors won't do anything until the corresponding slider is _moved_.
+This is a safety feature to prevent sudden motor movements after switch is enabled.
```

Comment:
I'm going to leave "to prevent sudden motor movements after switch is enabled" but I think it is "redundant". 

Why? Because the less people have to read the better. It is necessary to know that the actuators won't move unless the sliders move in this case. It is "good" to know that it is intentional behaviour. The extra detail of to prevent sudden motor movements is easily inferred.

It is "correct" though.

Reply:
I agree! It's redundant :disappointed:, now that I reflect back on it! I just remembered Beat telling me the reason for this safety feature, and I thought it wouldn't be so easily inferrable, but I guess it is :wink: